docs/node/run-your-node/prerequisites/set-up-trusted-execution-environment-tee.md

```diff
@@ -186,6 +122,24 @@ Architectural Enclaves is through **Application Enclave Service Manager
 can facilitate various SGX services such as launch approval, remote attestation
 quote signing, etc.
 
+Oasis node supports the (legacy) EPID and (newer) PCS attestation methods.
```

Comment:
```suggestion
Oasis Core supports the (legacy) EPID and (newer) DCAP attestation methods.
```

It is IAS vs. PCS (attestation services) or EPID vs. DCAP/ECDSA. Maybe DCAP is more well-known than PCS so let's switch to DCAP everyhwere.